doc/cookbook/splitting.md

```diff
@@ -66,3 +66,258 @@ $ pachctl put-file users master -c -f user_data.json --split json --target-file-
 # bytes chunk into the split files.
 $ pachctl put-file users master -c -f user_data.txt --split line --target-file-bytes 100
 ```  
+
+## Specifying Header/Footer
+
+Additionally, if your data has a common header or footer you can specify these manually. This is helpful for CSV data.
+
+You'll need to specify the header/footer on the parent directory of your data. Here we have an example of splitting a CSV w a header, then setting the header explicitly. Notice that once we've set the header, whenever we get a file under that directory, the header is applied. You can still use glob patterns to get all the data under the directory, and in that case the header is still applied.
+
+```
+$ cat users.csv 
+id,name,email
+4,alice,aaa@place.com
+7,bob,bbb@place.com
+$ cat users.csv | tail -n +2 | pc put-file bar master users --split line
+Reading from stdin.
+$ pachctl list-file bar master
+NAME  TYPE SIZE 
+users dir  42B  
+$ pachctl list-file bar master /users/
+NAME                    TYPE SIZE 
+/users/0000000000000000 file 22B  
+/users/0000000000000001 file 20B  
+$ pachctl get-file bar master /users/0000000000000000
+4,alice,aaa@place.com
+$ cat users.csv | head -n 1 | pc put-header bar master users 
+$ pachctl get-file bar master /users/0000000000000000
+id,name,email
+4,alice,aaa@place.com
+$ pachctl get-file bar master /users
+id,name,email
+$ pachctl get-file bar master /users/**
+id,name,email
+4,alice,aaa@place.com
+7,bob,bbb@place.com
+```
+
+For more info see `pachctl put-header --help`
```

Comment:
I assume the `pachctl put-header --help` docs are in the other PR. I'm going to review that one piece as well